ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/EthSignerBaseCommand.java

```diff
@@ -51,12 +56,19 @@
     footer = "EthSigner is licensed under the Apache License 2.0")
 public class EthSignerBaseCommand implements Config {
 
+  @SuppressWarnings("FieldMayBeFinal")
   @Option(
-      names = "--downstream-http-tls-known-servers-file",
-      description = "Path to a file containing the fingerprints of authorized servers",
-      arity = "1",
-      required = false)
-  private File downstreamKnownServersFile;
+      names = {"--chain-id"},
+      description = "The Chain Id that will be the intended recipient for signed transactions",
```

Comment:
I really don't understand the meaning here, but it could definitely be a knowledge gap issue. 
Is this replacing downstream-http-tls-known-servers-file?